taiga/models/models.py

```diff
@@ -980,6 +997,25 @@ class Project(InstanceResource):
         'us_statuses': UserStoryStatuses
     }
 
+    def get_item_by_ref(self, ref):
+        response = self.requester.get(
+            '/resolver?project={project_id}&ref={task_ref}',
+            task_ref=ref,
```

Comment:
just minor thing: can you rename this as ``obj_ref`` or anything more general than ``task_ref``?